usagov/vote-gov-drupal
# VOTE-856 update to wysiwyg embedded icon list

Update content editing permissions



## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-856

## Description

Add new function to wysiwyg placeholder to embed an icon from a theme file.

## Deployment and testing

### Post-deploy

1. run `lando retune`

### QA/Test

1. visit https://vote-gov.lndo.site/node/add/page
2. Click on the "embedded content" button in the wysiwyg and select "Placeholder" from the dropdown
3. Enter "icon:Printer" in the field and submit the dialog
4. Check and see if the icon shows up in the wysiwyg.
5. Save the page and check that it displays correct.

## Checklist for the Developer




## Checklist for the Peer Reviewers
